### PR TITLE
Pagetree integration tests

### PR DESCRIPTION
**Pull request by vxsx**
Some of the checks will be implemented separately (it's a bit unwieldy as it is)
It includes fixes for pagetree changes in regards to tests
also fixes small bug for pagetree where paste helpers were incorrectly shown even when clicked

### Musts

- [x] Shows a message when there are no pages
- [x] Pages and sub pages can be added through page tree
- [x] Move nodes around 
- [x] Nest pages
- [x] Unnest pages
- [x] Page cannot be published unless it has title/slug 
- [x] Cannot publish translation that doesn't have a title / slug
- [x] Can publish translation that does have a title / slug
- [x] Check that copy works
    - [x] Click on copy adds "paste buttons"
    - [x] Second click on copy of same node removes "paste buttons"
    - [x] Copy options dialog shows up and copy can be canceled
    - [x] Copy page into itself works
    - [x] Copy page into it's child works
    -  [x] Copy page into a parent works
    - [x] Copy page into a sibling works
    - [x] Copy page into root works
- [ ] Check that cut works
- [x] Opening and closing of nodes should work (tested implicitly through helper)
- [ ] After reload, current state is stored (open / closed nodes)
- [x] Check that delete removes a page - implicitly tested throughout the whole test suite
- [ ] Test search and filters (drag&drop should be disabled then)

### CODE REVIEW

**Comment by vxsx** [comment]
travis passes, but it's a false positive :(

File: cms/tests/frontend/integration/pagetree.js [hunk 1]
```diff
@@ -0,0 +1,904 @@
...
+    };
+};
+
+/**
+ * @function createJSTreeXPathFromTree
+ * @param {Object[]} tree tree object, see example
+ * @param {Object} [opts]
+ * @param {Object} [opts.topLevel=true] is it the top level?
+ * @example tree
+ *
```
Comment (FinalAngel) [inline cms/tests/frontend/integration/pagetree.js]: why the extra space?

File: cms/tests/frontend/integration/pagetree.js [hunk 2]
```diff
@@ -0,0 +1,904 @@
...
+            .waitForSelector('#page_form', function () {
+                this.sendKeys('#id_title', 'Nested page');
+                this.click('input[name="_save"]');
+            })
+            .waitUntilVisible('.success')
+            .then(cms.expandPageTree())
+            .then(function () {
+                var pageId = cms.getPageId('Nested page');
+                test.assertExists(
+                    xPath(createJSTreeXPathFromTree([{ name: 'Homepage', children: [{ name: 'Nested page' }] }])),
```
Comment (FinalAngel) [inline cms/tests/frontend/integration/pagetree.js]: would add the objects on new lines for better readability

**Comment by FinalAngel** [comment]
LGTM

File: cms/tests/frontend/integration/pagetree.js (hunk 1 above)
Comment (vxsx) [inline cms/tests/frontend/integration/pagetree.js]: because of how examples are processed by yuidoc - it has to be valid markdown

**Pull request merged**